### DISCUSSION TITLE
TYPE OF VOTING

### DISCUSSION THREAD

**Discussion by toney010319** [Group-1-hackathon/voting-system#18] [Q&A] [answered]
btw guys yung type of voting ba naten is choose one prefered candidate or Rubrix type [(UI/UX, Structure etc, candidate with highest grade will win]  ?

**Comment by devpapi0891** [comment]
Parang ganto na lang para simple lang 

![image](https://github.com/Group-1-hackathon/voting-system/assets/97945953/dbf38af8-4427-4009-b6bc-ee9bff937cb1)

**Reply by devpapi0891** [reply, marked as answer]
Lets say nasa page na tayo for specific event/:id
Meron don vote now button that will trigger open a modal form that lists candidates. Then they can just click one then submit. Once submitted. If goods as into, mag redirect sila into a specific page congratulating them (your vote is submitted) Pag balik nila sa page ng event/:id yung vote now magiging "You voted for this event" Parang ganon.

**Comment by wilmer162000** [comment]
While the Rubrix-type of voting can provide specific inputs from each voter, it may complicate matters, especially considering the limited amount of time we have available. **It might be better to simplify it to one person = one vote**. We can leave the more complex aspects to the senior judges. Additionally, managing a scale of more than a hundred votes could become a nightmare and would likely require automation programs.
